packages/kuma-gui/src/types/api.d.ts

```diff
@@ -23,6 +23,7 @@ export interface PaginatedApiListResponse<ResourceType> extends ApiListResponse<
 export interface PaginationParameters {
   size?: number
   offset?: number
+  filter?: Record<string, string>
```

Comment:
Not really a "pagination" parameter, but seeing as this handwritten type will go soon, I didn't bother worrying about the naming weirdness too much.